src/index.ts

```diff
@@ -1,17 +1,59 @@
 import { Command } from 'commander';
 import { description, version } from '../package.json';
-import commands from './commands';
+
+function isObject(value: unknown): value is object {
+  return typeof value === 'object';
+}
+
+function isFunction(value: unknown): value is () => unknown {
+  return typeof value === 'function';
+}
+
+async function getFunctionByName(functionName: string): Promise<() => unknown> {
+  const { faker } = await import('@faker-js/faker/locale/en');
+  for (const module of Object.values(faker)) {
+    if (!isObject(module)) {
+      continue;
+    }
+
+    for (const entry of Object.values(module)) {
+      if (!isFunction(entry)) {
+        continue;
+      }
+
+      // we need to use "includes" since the function names are "bound functionName"
+      if (!entry.name.includes(functionName)) {
+        continue;
+      }
+
+      return entry;
+    }
+  }
+
+  throw new Error(
+    `You tried to generate a value with the function name "${functionName}", but this name does not exist in Faker. Please checkout their documentation (https://fakerjs.dev) to look for available functions.`,
```

Comment:
I'm not sure about this one:

```suggestion
    `You tried to generate a value with a function named "${functionName}", but it does not exist in Faker. Please checkout their documentation (https://fakerjs.dev) to look for available functions.`,
```

Reply:
This has been moved to the `ReferenceError` class.